packages/mantine/src/styles/Pagination.module.css

```
@@ -1,3 +1,8 @@
+.root {
+    --pagination-active-bg: var(--mantine-primary-color-light);
+    --pagination-active-color: var(--mantine-primary-color-5);
```

Since the 5 can change:
```suggestion
    --pagination-active-color: var(--mantine-primary-color-filled);
```

---

👍